libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationPresenter.java

```diff
@@ -23,10 +26,15 @@ void onRecenterClick() {
     view.setSummaryBehaviorHideable(false);
     view.setSummaryBehaviorState(BottomSheetBehavior.STATE_EXPANDED);
     view.updateWaynameVisibility(true);
-    view.resetCameraPosition();
+    onResetCameraAndArrow();
```

Comment:
Here when you say `Arrow` you mean the _puck_? If so, we should update. That's how we name it internally.